lib/firmwares.json

```diff
@@ -24,7 +24,8 @@
       "repo": "url",
       "npm": {
           "package": "node-pixel",
-          "version": "^0.4.1"
+          "repo": "git+https://github.com/ajfisher/node-pixel",
```

Comment:
Yeah that's probably a good idea. I'm very close to 0.8 release too.

On Sun, 11 Sep 2016, 11:30 Steve Mao <notifications@github.com> wrote:

> In lib/firmwares.json
> <https://github.com/johnny-five-io/nodebots-interchange/pull/20#discussion_r78286550>
> :
>
> > @@ -24,7 +24,8 @@
> >        "repo": "url",
> >        "npm": {
> >            "package": "node-pixel",
> > -          "version": "^0.4.1"
> > +          "repo": "git+https://github.com/ajfisher/node-pixel",
>
> Do you want to bump it instead?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/johnny-five-io/nodebots-interchange/pull/20/files/3246a19af22d3d3540b037b767e76c2a33bd9642#r78286550>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAHRoxw7_DtE7nFpW0TEfbnj8an5PZyEks5qo1mhgaJpZM4J51tc>
> .
>